point_viewer_grpc/src/service.rs

```diff
@@ -124,7 +124,7 @@ fn stream_points_back_to_sink(
 
                 reply_size += bytes_per_point;
                 if reply_size > max_message_size - bytes_per_point {
-                    tx.send((reply.clone(), WriteFlags::default())).unwrap();
+                    tx.send((reply.clone(), WriteFlags::default())).unwrap_or(());
```

Comment:
common way to ignore errors is `let _ = tx.send()...`

But I am worried that this continues the loop - once send fails, this thread should stop its work.

Reply:
I hadn't thought about that. Should I make the callback return a Result?

Reply:
sure.